Crypto/Crypto/Presentation/Views/LoginView/LogInViewModel.swift

```diff
@@ -34,7 +34,7 @@ class LogInViewModel: ObservableObject{
                             break
                         case .failure(let error):
                             DispatchQueue.main.async {
-                                self?.state = (self?.state.clone(withIsLoggedIn: false, withError: error.localizedDescription, withAlert: true))!
+                                self?.state = (self?.state.clone(withIsLoggedIn: false,withMessage: "email or password invalid", withAlert: true))!
                             }
```

Comment:
I would also suggest moving these error messages to a separate file, that way if you need to modify any you have them all in the same place and you don't have to search through the code